File: app/src/main/java/com/vegashot/vegashot/model/ClassSchedule.java
package com.vegashot.vegashot.model;

import java.io.Serializable;
import java.util.List;

public class ClassSchedule implements Serializable {
    public final List<DailySchedule> dailySchedules;

    public ClassSchedule(List<DailySchedule> dailySchedules) {
        this.dailySchedules = dailySchedules;
    }
}

File: app/src/main/java/com/vegashot/vegashot/model/ClassSchedule.kt
package com.vegashot.vegashot.model

import java.io.Externalizable
import java.io.ObjectInput
import java.io.ObjectOutput
import java.io.Serializable

public class ClassSchedule(var dailySchedules: List<DailySchedule>) : Externalizable {
    override fun readExternal(input: ObjectInput) {
        dailySchedules = input.readObject() as List<DailySchedule>
    }

    override fun writeExternal(output: ObjectOutput) {
        output.writeObject(dailySchedules)
    }
}
